doc/usage.md

```diff
@@ -187,7 +187,10 @@ total_query_count
 :   Total number of SQL commands pooled by **pgbouncer**.
 
 total_server_assignment_count
-:   Total times a server was assigned to a client
+:   Total times a server was assigned to a client.
+
+total_server_closed_count
```

Comment:
I think counting the amount of server opens/creations/connections/connection establishments (naming tbd) sounds a bit more natural than how many were closed.